x-pack/plugins/security_solution/public/timelines/components/timeline/header/save_timeline_button.tsx

```diff
@@ -56,27 +53,28 @@ export const SaveTimelineButton = React.memo<SaveTimelineButtonProps>(({ timelin
   } = useDeepEqualSelector((state) => getTimelineStatus(state, timelineId));
 
   const isSaveButtonMounted = useIsElementMounted(SAVE_BUTTON_ELEMENT_ID);
-  const [hasSeenTimelineSaveTour, setHasSeenTimelineSaveTour] = useLocalStorage({
-    defaultValue: false,
+  const [timelineTourStatus, setTimelineTourStatus] = useLocalStorage({
+    defaultValue: { isTourActive: true },
     key: LOCAL_STORAGE_KEY,
+    isInvalidDefault: (valueFromStorage) => {
```

Comment:
@logeekal The key in that file is prefixed with `securitySolution` and this one isn't. The prefix is added by `useLocalStorage`. So sadly we have to duplicate it here...

Reply:
@logeekal Sure, that would be ideal but this piece of code makes this pretty hard without changing the hook in this PR:

https://github.com/elastic/kibana/blob/b55dae32f67a5cd2fb106c57a9712f443e7492f2/x-pack/plugins/security_solution/public/common/components/local_storage/index.tsx#L33

Reply:
Understood. in that case I think it is okay to go ahead. I am, anyway, creating a new timeline tour already, which will override the current tour. 

But since that is multistep tour, I am thinking of using same strategy `Guided Rule Tour` takes as in below file by plugging in guidedOnboardingApi:

https://github.com/elastic/kibana/blob/b55dae32f67a5cd2fb106c57a9712f443e7492f2/x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/alternative_tour/tour.tsx#L58-L60